semantics/src/main/kotlin/com/strumenta/kolasu/semantics/symbol/description/SymbolDescription.kt

```diff
@@ -0,0 +1,39 @@
+package com.strumenta.kolasu.semantics.symbol.description
+
+data class SymbolDescription(
```

Comment:
I think we should add a comment to indicate that they can represent nodes or builtin elements. And the fact that they represent a "slim" version of nodes, to be used in the Symbol Resolution Index

Reply:
I agree, the current implementation is not really well documented as the main focus consisted in experimenting first. I will add a detailed explanation of every component once done, thanks.